src/applications/vaos/services/healthcare-services/index.js

```diff
@@ -0,0 +1,33 @@
+import { getAvailableClinics } from '../../api';
+import { transformAvailableClinics } from './transformers';
+import { mapToFHIRErrors } from '../../utils/fhir';
+import { getLocation } from '../location';
+
+/*
+ * This is used to parse the fake FHIR ids we create for organizations
+ */
+function parseId(id) {
+  return id.replace('var', '');
+}
+
+export async function getAvailableHealthCareSystems(
```

Comment:
I would call this something like `getAvailableHealthcareServices`, to be consistent with the resource name.

Reply:
Done.